### DISCUSSION TITLE
Issue when stack invalid FX VST is loaded with other FX VST (video)

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#314] [Help]
Happy New Year!! 
There is an issue where an older version of Waves VST was loaded but not it was inactive (because upgrade and not matching VST ID with newer, this is another issue) together with additional VST. It seems the AG plugin stack will keep track of the listing/IDs based on active VST.
I have attached a video to better explain this issue.

https://user-images.githubusercontent.com/68957086/104485809-f6446700-5598-11eb-9174-7e4fd16a08a2.mp4

**Comment by CaBuffalo** [comment]
Will AG work properly if you delete the old insert?

**Reply by jaacMusic** [reply]
I tried deleting the bad insert and that didn't help. The workaround was to as to what Andreas mentioned, which is to remove the bad insert before adding any new insert in the chain.

**Comment by apohl79** [comment]
yes, the server does not “track” the first slot correctly, as the plugin fails to load. the plugin should probably tell the server the insert id to makes sure both are in sync in edge cases like this. but as @CaBuffalo suggested, removing the broken plugin first (before adding new inserts) will make it work.
